index.html

```diff
@@ -4,10 +4,58 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet">
     <link rel='stylesheet' href='./style.css' />
     <title>Component Library</title>
 </head>
 <body>
+
+    <h1>Badges</h1>
+    <h3>notification badges</h3>
+    <div class='badge-size-1 badge-green border-radius-3'></div>
+    <div class='badge-size-1 badge-red border-radius-3'></div>
+    <div class='badge-size-1 badge-yellow border-radius-3'></div>
+    <div class='badge-size-1 badge-violet border-radius-3'></div>
+    <br/>
+    <div class='badge-size-2 badge-green border-radius-3'></div>
+    <div class='badge-size-2 badge-red border-radius-3'></div>
+    <div class='badge-size-2 badge-yellow border-radius-3'></div>
+    <div class='badge-size-2 badge-violet border-radius-3'></div>
+    <br/>
+    
+    <h3>text only badges</h3>
+    <div class='badge-size-1 badge-text-1 flex-center badge-padding-1 border-radius-3 badge-violet'>
+        4
+    </div><br/>
+
+    <h3>badges on icon</h3>
+    <div class='inline-block pos-relative'>
+        <span class='material-icons'>account_balance</span>
+        <div class='badge-size-1 badge-text-1 flex-center border-radius-3 badge-violet pos-absolute top-left-60'>1</div>
+    </div>
+    <div class='inline-block pos-relative'>
+        <span class='material-icons'>account_balance</span>
+        <div class='badge-size-1 badge-padding-1 badge-text-1 flex-center border-radius-3 badge-violet pos-absolute top-left-60'>1</div>
+    </div><br/><br/><br/>
```

Comment:
You can avoid using <br>. Instead margins can be used. (Personal Opinion though:) )

Reply:
yes I wont use br tags in the components or documentation. this was just for this particular html doc only.

Reply:
i have removed the br tags.